app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MessagesController < ApplicationController
+  before_action :set_message, only: %i[show update destroy]
+
+  # GET random message
+  def index
+    @message = Message.order('RANDOM()').first
```

Comment:
* Kindly address the cors error i get when i try to run your project HINT 💭 you can fix it by installing 
* `gem "rack-cors"`  and adding this code in cors.rb file 
```
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
  allow do
    origins "your frontend url"

    resource "*",
      headers: :any,
      methods: [:get, :post, :put, :patch, :delete, :options, :head]
  end
end
```
![image](https://github.com/jmonto55/hello-react-frontend/assets/52497246/3071f665-56b6-419f-8c7f-09d9d4240bc8)

Reply:
Thank you for the hint ! @Davidosky007

Reply:
This change has been done well